pkg/store/consul/statusstore/rcstatus/status.go

```diff
@@ -0,0 +1,38 @@
+package rcstatus
+
+import (
+	"encoding/json"
+
+	"github.com/square/p2/pkg/store/consul/statusstore"
+	"github.com/square/p2/pkg/types"
+	"github.com/square/p2/pkg/util"
+)
+
+type RCStatus struct {
+	NodeTransfer NodeTransfer `json:"node_transfer"`
```

Comment:
I wonder if we want to make this a pointer or a list so it's more obvious when it's empty? otherwise we can check that `OldNode == "" && NewNode == ""`

Reply:
Yeah thats a good question. I think it depends on how we end up using it. Will the RCStatus always exist and the we'll check to see if NodeTransfer is empty, or will we delete the RCStatus when we're done so we'll be checking for a `NoStatus` error from Consul to see if a node transfer is not in progress?

Reply:
Once an RC status record is created I'd think it should exist for the whole lifetime of the RC, and we'll just empty out the `NodeTransfer` part when the transfer is done. The question is just whether the `NodeTransfer` structure itself should be `nil` (which is only allowed in Go if you use a pointer) or if it should have `""` for both of its fields

Reply:
Okay in that case I agree that we should use a pointer